src/components/Products/ProductsPagination/ProductsPagination.tsx

```diff
@@ -0,0 +1,81 @@
+import { DEFAULT_MIN_PAGE } from '../../../utils/constants';
+import { getMaxPage } from '../../../utils/utils';
+import styles from './ProductsPagination.module.scss';
+
+export function ProductsPagination(props: {
+  currentPage: number;
+  handlePageChange: (page: number) => void;
+  handleLimitChange: (limit: number) => void;
+  itemsPerPage: number;
+  totalItems: number;
+  searchLimit: number;
+}) {
+  const {
+    currentPage,
+    handlePageChange,
+    itemsPerPage,
+    totalItems,
+    searchLimit,
+    handleLimitChange,
+  } = props;
+
+  const prevPageHandler = () => {
+    if (currentPage !== DEFAULT_MIN_PAGE) {
+      let newPage = currentPage - 1;
+      newPage = newPage < DEFAULT_MIN_PAGE ? DEFAULT_MIN_PAGE : newPage;
+      handlePageChange(newPage);
+    }
+  };
+
+  const nextPageHandler = () => {
+    const maxPage = getMaxPage(itemsPerPage, totalItems);
+    if (currentPage !== maxPage) {
+      let newPage = currentPage + 1;
+      newPage = newPage > maxPage ? maxPage : newPage;
+      handlePageChange(newPage);
+    }
+  };
+
+  const inputLimitChangeHandler = (
+    event: React.ChangeEvent<HTMLSelectElement>
+  ) => {
+    event.preventDefault();
+    const limit = +event.target.value;
+    handleLimitChange(limit);
+    handlePageChange(DEFAULT_MIN_PAGE);
+  };
+
+  return (
+    <div className={styles.pagination}>
+      <div className={styles.limitContainer}>
+        <label htmlFor="limits">Choose a products per page limit:</label>
```

Comment:
one tip from me: if you wrap your input in label, and then you don't need 'for' and 'id', it will works fine